src/test/packages/typedoc-single-package-example/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typedoc-single-package-example",
+  "version": "1.0.0",
+  "description": "An example of using typedoc with a single package",
+  "private": true,
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "tsc",
+    "test": "node ."
+  },
+  "devDependencies": {
+    "typescript": "^4.2.4"
```

Comment:
Technically this package should depend on typedoc? I never had to install it because the tests bring their own typedoc.